Basics Practice/problem6.py

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
A similar example is present in this repo at https://github.com/Rishit-dagli/Python3-by-practice/tree/master/Training%20Modules/01%20Introduction/1.07%20Print and I don't feel this adds anything new